app/views/audit_mailer/summary.html.erb

```diff
@@ -1,20 +1,23 @@
-<body style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; color: #3f3f3f; background-color: #f1f1f1; padding: 10px 24px">
-<h2 style="font-weight: normal; text-transform: uppercase; font-size: 120%;"><%= _("<b>Foreman</b> audit summary").html_safe %></h2>
-<ul style="list-style-type: none;">
+<head>
```

Comment:
To prevent duplicating this into each mailer, add a layout that's used for everything containing the standard html/head/stylesheet/body headers/footers.